File: QE_FST_Selenium_java/Activity7_ElementsEnabled.java
package demos;

import org.openqa.selenium.By;
import org.openqa.selenium.WebDriver;
import org.openqa.selenium.firefox.FirefoxDriver;

public class Activity7_ElementsEnabled {
    public static void main(String[] args) {
        
        WebDriver driver=new FirefoxDriver();
        driver.get("https://training-support.net/webelements/dynamic-controls");
        System.out.println(driver.getTitle());
        if(driver.findElement(By.id("textInput")).isEnabled())
        {
            System.out.println("Text Input is Enabled");
        }
        else{ System.out.println("Text Input is Disabled");}
        driver.findElement(By.id("textInputButton")).click();
        if(driver.findElement(By.id("textInput")).isEnabled())
        {
            System.out.println("Text Input is Enabled");
        }
        else{ System.out.println("Text Input is Disabled");}
        driver.quit();
    }
}

File: QE_FST_Selenium_python/Activity7_ElementsEnabled.py
from selenium import webdriver
from selenium.webdriver.common.by import By
with webdriver.Firefox() as driver:
    driver.get("https://training-support.net/webelements/dynamic-controls")
    print(driver.title)
    textInput=driver.find_element(By.ID,"textInput")
    if(textInput.is_enabled()):
        print("TextInput is Enabled")
    else:
        print("TextInput is not Enabled")
    textInputButton=driver.find_element(By.ID,"textInputButton")
    textInputButton.click()
    if(textInput.is_enabled()):
        print("TextInput is Enabled")
    else:
        print("TextInput is not Enabled")
